### PR TITLE
Phillyjug blog

### PR DESCRIPTION
**Pull request by ajm01**

### CODE REVIEW

File: posts/2021-10-05-openliberty-jug-presentation.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+layout: post
+title: "A Fun Cup of Joe - Find Out Why Open Liberty is Your Next Cloud Native Java Runtime"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ajm01
+author_github: https://github.com/ajm01
+seo-title: A Fun Cup of Joe - Find Out Why Open Liberty is Your Next Cloud Native Java Runtime
+seo-description: A Fun Cup of Joe - Find out why Open Liberty is Your Next Cloud Native Java Runtime
+blog_description: "Find out why Open Liberty is Your Next Cloud Native Java Runtime. A recap of a presentation and demos at the Philly JUG (Philadelphia Java User Group)."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
```
Comment (dmuelle) [inline posts/2021-10-05-openliberty-jug-presentation.adoc]: for visibility in social media previews, replace the twitter card with the image from the post:

https://openliberty.io/img/blog/cupofjoe.png

File: posts/2021-10-05-openliberty-jug-presentation.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+---
+= A Fun Cup of Joe - Find Out Why Open Liberty is Your Next Cloud Native Java Runtime
+Andrew Mauer <https://github.com/ajm01>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+== Why Open Liberty?
+As the popularity of a microservice architecture and cloud-native application development experience grows, it is an exciting time to talk about the Open Liberty framework's support of those. I recently had the opportunity to lead a discussion at a meeting of the Philadelphia Java User Group (JUG) meetup where I highlighted the latest features and design ideas contained within Open Liberty. As part of that discussion I ran two demos where I created and interacted with an Open Liberty-hosted RESTful web service as an application developer - both in a local developer environment and live in a remote Kubernetes cluster environment.
```
Comment (dmuelle) [inline posts/2021-10-05-openliberty-jug-presentation.adoc]: recommend starting with the second sentence- it makes a more direct connection. You could either move the first sentence to the end of the paragraph, or omit it.

File: posts/2021-10-05-openliberty-jug-presentation.adoc [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+image::img/blog/cupofjoe.png[A Fun Cup Of Joe, align="center"]
+
+The focus of the presentation centered around Open Liberty as an open framework for Java developers that highly optimizes the development of microservices using Eclipse MicroProfile and Jakarta EE APIs. To highlight Open Liberty as a flexible, fast and lightweight Java runtime, the discussion touched on important Open Liberty framework design attributes such as its simple configuration to enable "Just Enough Runtime" - configuring the framework to enable only the subset of features an application needs at runtime, its low operating cost due to a reduced memory footprint based on that minimal configuration, zero migration, Kubernetes optimizations and its efficient rapid developer experience. Additionally, the fact that Open Liberty supports the deployment of Spring Boot applications generated an enthusiastic question from a member of the audience interested in understanding how Open Liberty could help a shop with hundreds of Spring Boot applications become more efficient.
```
Comment (dmuelle) [inline posts/2021-10-05-openliberty-jug-presentation.adoc]: a couple suggestions here-

for concision and clarity:
The focus of the presentation centered around Open Liberty as an open framework for Java developers that highly optimizes the development of microservices using Eclipse MicroProfile and Jakarta EE APIs
--->
The presentation focused  how Open Liberty highly optimizes the development of microservices by using Eclipse MicroProfile and Jakarta EE APIs

The sentence that follows is really long and a bit hard to parse- what are the design attributes and what are the descriptions of those attributes- they all kind of jumble together. Consider breaking this sentence up, shortening it, or converting to a bulleted list.

File: posts/2021-10-05-openliberty-jug-presentation.adoc [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
...ght inner-loop experience provided by the Liberty Maven Plugin. By demonstrating a Java RESTful application being enhanced locally in an IDE, the participants were able to see the quick turnaround time developers can achieve using the https://openliberty.io/docs/21.0.0.10/development-mode.html[dev mode] feature of the Liberty Maven Plugin. Code and configuration updates were shown to take place immediately in a local running Open Liberty runtime instance, managed by the Liberty Maven Plugin on behalf of the developer with test cases running automatically providing immediate developer feedback. Additionally, Open Liberty's Kubernetes integration points were demonstrated showing that same developer inner-loop experience occurring in a remote development cluster hosted in a Red Hat OpenShift environment. Participants were able to observe in real-time the effect of an application code update being reflected automatically in a remote Kubernetes cluster using the OpenShift Do (odo) tool and the java-openliberty development https://github.com/OpenLiberty/application-stack[stack] in conjunction with the same Liberty Maven Plugin to manage the OpenShift interactions on the developer's part.
```
Comment (dmuelle) [inline posts/2021-10-05-openliberty-jug-presentation.adoc]: is it possible to combine the description of the demo with the list of links? For each link you could move the part of the paragraph that follows it, eg.

https://youtu.be/qvoEhgw90Ig[Demo 1 - Developing locally with Open Liberty's dev mode]:: 
By demonstrating a Java RESTful application being enhanced locally in an IDE, the participants were able to see the quick turnaround time developers can achieve using the https://openliberty.io/docs/21.0.0.10/development-mode.html[dev mode] feature of the Liberty Maven Plugin.

If you use the description list syntax show above (double colon :: after the term, description on the following line) it should apply a nice heading style to the links and nest the descriptions neatly beneath.

also might be good to include a link to [the Open Liberty Maven plugin](https://github.com/OpenLiberty/ci.maven) on first mention in this section

File: posts/2021-10-05-openliberty-jug-presentation.adoc [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+Feel free to check out video segments of the discussion at the following links
+
+* https://youtu.be/h-OrANJInnk[Open Liberty Overview]
+* https://youtu.be/qvoEhgw90Ig[Demo 1 - Developing locally with Open Liberty's dev mode]
+* https://youtu.be/nX3M04zNNNw[Open Liberty and Kubernetes Optimizations]
+* https://youtu.be/vTjwaersm-0[Demo 2 - Developing live in a Kubernetes environment with Open Liberty]
+* https://youtu.be/Dv063vQsYJY[Open Liberty Summary]
+
+The Philadelphia JUG meetup presentation was well attended with 35+ attendees joining via the meetup's web conferencing tool. The host of the meetup was excited to have the group listen to the strengths of Open Liberty and its developer's experience and I was happy to have had the opportunity to speak with this group about the exiting attributes of Open Liberty. If your group is interested in hearing more, please feel free to reach out to https://twitter.com/yeekangc[YK and team] to schedule a similar discussion at any time.
```
Comment (dmuelle) [inline posts/2021-10-05-openliberty-jug-presentation.adoc]: for concision and clarity:

The host of the meetup was excited to have the group listen to the strengths of Open Liberty and its developer's experience and I was happy to have had the opportunity to speak with this group about the exiting attributes of Open Liberty.
--->
The host of the meetup was excited for the group to hear about the strengths of Open Liberty and its developer experience, and I was happy for the opportunity to speak with them.

**Review by dmuelle — CHANGES_REQUESTED** [review]
ID Review- great post! I just made a few suggestions to break up the "wall of text" a  bit and make a couple sentences more concise and readable. Let me know if you have any questions- thanks!

File: posts/2021-10-05-openliberty-jug-presentation.adoc (hunk 4 above)
Comment (ajm01) [reply posts/2021-10-05-openliberty-jug-presentation.adoc]: I couldn't get the '::' to work - i used a '+' to include paragraph text under the given bullet point.

**Review by dmuelle — APPROVED** [review]
